d20/Manual/BattleMap.py

```diff
@@ -6,15 +7,22 @@
 import threading
 import pathlib
 import re
+import sys
 
-from d20.Manual.Exceptions import (DuplicateObjectError)
-from d20.Manual.Facts import (RegisteredFacts, loadFact)
+from d20.Manual.Exceptions import (DuplicateObjectError, NotFoundError)
+from d20.Manual.Facts import (Fact, RegisteredFacts, loadFact)
 
 from d20.Manual.Temporary import (
     TemporaryObjectOnDisk,
     TemporaryObjectStream
 )
 
+from typing import (ItemsView, List, Dict, Type,
+                    Optional, Set, Union, Match,
+                    Iterator, ValuesView, TypeVar)
```

Comment:
fix import ordering, and use proper whitespace between imports and global variable declarations.